src/compliance/models/compliance-status.model.ts

```diff
@@ -0,0 +1,19 @@
+/* istanbul ignore file */
+
+import { ApiProperty } from '@nestjs/swagger';
+
+import { FromBigNumber } from '~/common/decorators/transformation';
+
+export class ComplianceStatusModel {
+  @ApiProperty({
+    description: 'Are the compliance requirements paused',
+    type: 'boolean',
+    example: true,
+  })
+  @FromBigNumber()
+  readonly arePaused: boolean;
```

Comment:
```suggestion
  @ApiProperty({
    description: 'Indicator to know if compliance requirements are paused or not',
    type: 'boolean',
    example: true,
  })
  readonly paused: boolean;
```

Reply:
`@FromBigNumber` is only needed when return type is BigNumber. Feels like it got copied over